cluster/manifests/skipper/rbac.yaml

```diff
@@ -5,12 +5,26 @@ metadata:
   namespace: kube-system
   labels:
     application: skipper-ingress
+    component: ingress
 {{ if eq .Cluster.ConfigItems.skipper_open_policy_agent_enabled "true" }}
   # Note: if the role extends beyond OPA use, this condition can be removed
   annotations:
     iam.amazonaws.com/role: "{{ .Cluster.LocalID }}-app-skipper-ingress"
 {{ end }}
 ---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
Why do we need this instead of existing ServiceAccount (see also #6999)?

Reply:
Lets also split such changes in two PRs

Reply:
because least privileges and you broke OPA

The problem of https://github.com/zalando-incubator/kubernetes-on-aws/pull/6999 is that if "exec", then skipper-ingress does not use serviceaccount skipper-ingress which has https://github.com/zalando-incubator/kubernetes-on-aws/blob/beta/cluster/manifests/skipper/rbac.yaml#L8-L12 that if OPA is enabled leads to no permission to fetch the policy from S3.

Reply:
Existing `skipper-ingress` ServiceAccount permissions should be reduced to minimum necessary to fetch the policy from S3.

Reply:
Lets revert #6999 https://github.com/zalando-incubator/kubernetes-on-aws/pull/7131 and work from there

Reply:
> Existing `skipper-ingress` ServiceAccount permissions should be reduced to minimum necessary to fetch the policy from S3.

no we should not change the running serviceaccount that is assigned to a workload!